src/GraphQL.Conventions/Types/Resolution/ObjectReflector.cs

```diff
@@ -27,6 +27,8 @@ class ObjectReflector
 
         private readonly MetaDataAttributeHandler _metaDataHandler = new MetaDataAttributeHandler();
 
+        internal List<string> IgnoredNamespaces { get; } = new List<string>() { nameof(System) + "." };
```

Comment:
What about making this a `HashSet<>`, that way you don't need to do the `if (!Contains) { Add() }` further down. :-)

Reply:
Also, this can be public since `ObjectReflector` is only seen within the library